mysql-test/mysql-test-run.pl

```diff
@@ -1091,6 +1091,7 @@ sub command_line_setup {
              'gdb'                      => \$opt_gdb,
              'client-gdb'               => \$opt_client_gdb,
              'manual-gdb'               => \$opt_manual_gdb,
+             'manual-lldb'              => \$opt_manual_lldb,
```

Comment:
--manual-lldb did not work on the 5.5 GCA branch. It may or may not work on the 5.5 trunk. I can remove this bit.